src/libs/fsdrv/lv_fs_littlefs.c

```diff
@@ -0,0 +1,165 @@
+#include "../../../lvgl.h"
+#if LV_USE_FS_LITTLEFS != '\0'
+
+#include <LittleFS.h>
+#include "../../core/lv_global.h"
+
+struct LittleFile {
+    File file;
+};
+
+/**********************
+ *  STATIC PROTOTYPES
+ **********************/
+static void fs_init(void);
+static void * fs_open(lv_fs_drv_t * drv, const char * path, lv_fs_mode_t mode);
+static lv_fs_res_t fs_close(lv_fs_drv_t * drv, void * file_p);
+static lv_fs_res_t fs_read(lv_fs_drv_t * drv, void * file_p, void * buf, uint32_t btr, uint32_t * br);
+static lv_fs_res_t fs_write(lv_fs_drv_t * drv, void * file_p, const void * buf, uint32_t btw, uint32_t * bw);
+static lv_fs_res_t fs_seek(lv_fs_drv_t * drv, void * file_p, uint32_t pos, lv_fs_whence_t whence);
+static lv_fs_res_t fs_tell(lv_fs_drv_t * drv, void * file_p, uint32_t * pos_p);
+
+/**
+ * Register a driver for the LittleFS File System interface
+ */
+void lv_port_littlefs_init(void)
+{
+    fs_init();
+
+    /*Add a simple drive to open images*/
+    static lv_fs_drv_t fs_drv;
```

Comment:
use global var for fs_drv, reference to fatfs.

lv_fs_drv_t * fs_drv_p = &(LV_GLOBAL_DEFAULT()->littlefs_fs_drv);

and define `littlefs_fs_drv`  it in `_lv_global_t` as below:

#if LV_USE_FS_LITTLEFS
    lv_fs_drv_t littlefs_fs_drv;
#endif